README.md

```diff
@@ -1,4 +1,134 @@
-# Trampoline V2
+# docker-ci-helper
 
-This is a shell script for running another script in a Docker
-container.
+This repository contains `trampoline_v2.sh`, a shell script for
+running another script in a Docker container. Primary use case of this
+shell script is to use it as a test driver on various CI systems. By
+utilizing Docker, your tests will be very portable so that you can run
+them on any CI systems as well as on your workstations with minimum
+efforts.
+
+## trampoline_v2.sh
+
+This script does 3 things.
+
+1. Prepare the Docker image for the test.
+2. Run Docker with appropriate flags to run the test.
+3. Upload the newly built Docker image.
+
+We'll discuss each steps in detail.
+
+### Prepare the Docker image
+
+1. Try to download a Docker image specified by `TRAMPOLINE_IMAGE`
+   environment variable.
+2. If `TRAMPOLINE_DOCKERFILE` environment variable is specified, it
+   will build the Docker image from that Dockerfile. If there's a
+   downloaded Docker image, Trampoline V2 uses the downloaded image as
+   a cache.
+
+We recomnend you have the Dockerfile in the repo and specify
+`TRAMPOLINE_DOCKERFILE`. This will allow you to create a single pull
+request containing changes in tests as well as changes in the
+Dockerfile.
+
+### Run Docker with appropriate flags to run the test
+
+* Mounting the source at /workspace
```

Comment:
```suggestion
* Mounting the current working directory at /workspace
```

Reply:
Actualy it's not the current working directory. Now it tries to find a git root and mount the whole git root.

Reply:
I clarified what will be mounted at /workspace.